packages/cli/generation/local-generation/local-workspace-runner/src/dynamic-snippets/csharp/DynamicSnippetsCsharpTestGenerator.ts

```diff
@@ -0,0 +1,115 @@
+import { mkdir, writeFile } from "fs/promises";
+import path from "path";
+
+import { Style } from "@fern-api/browser-compatible-base-generator";
+import { Config, DynamicSnippetsGenerator } from "@fern-api/csharp-dynamic-snippets";
+import { AbsoluteFilePath, RelativeFilePath, join } from "@fern-api/fs-utils";
+import { dynamic } from "@fern-api/ir-sdk";
+import { TaskContext } from "@fern-api/task-context";
+
+import { FernGeneratorExec } from "@fern-fern/generator-exec-sdk";
+
+import { convertDynamicEndpointSnippetRequest } from "../utils/convertEndpointSnippetRequest";
+import { convertIr } from "../utils/convertIr";
+
+const PROJECT_FILE_CONTENT = `
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <LangVersion>12</LangVersion>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+        <IsPackable>false</IsPackable>
+        <IsTestProject>true</IsTestProject>
+        <PolySharpIncludeRuntimeSupportedAttributes>true</PolySharpIncludeRuntimeSupportedAttributes>
```

Comment:
We should remove PolySharp and make it so that the language version is the one you want to target for compatibility.